internals/webpack/webpack.prod.babel.js

```diff
@@ -72,6 +77,26 @@ module.exports = require('./webpack.base.babel')({
 
       AppCache: false,
     }),
+
```

Comment:
This should be its own PR, it's larger than just a Webpack4 update

Reply:
You're talking about webpack-pwa-manifest? It's basically the reason for this PR and for almost all of the changes here... I wrote it in response to the post-merge comments on my webpack 4 PR (#2148). Perhaps my mistake was naming the PR "addendum" but it has almost no reason to be without webpack-pwa-manifest.

If you really want me to break it into 2 I would rather remove from this one everything which isn't related to webpack-pwa-manifest (small dependency upgrades and bundle size improvements). Lmk what you think!

Reply:
Got it, I messed with it a bit more and I agree now with why its in this PR. Disregard